src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static int inputCount = 0;
+    public static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String input;
+        Scanner in = new Scanner(System.in);
+
+
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?\n");
+
+        input = in.nextLine();
+
+        while (!(input.equals("bye"))) {
+
+            if (input.equals("list")) {
+                input = printList(in);
+
+            } else if (input.contains("done")) {
+                input = testTaskDone(input, in);
+
+            } else if (input.contains("todo")) {
+                try {
+                    testInput(input); // test todo input
+
+                    Task whatToDo = toDoMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatToDo + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+
+                    input = in.nextLine();
+                } catch (DukeExceptions e3) {
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
```

Comment:
Perhaps you could consider to extract these final strings (also found in other parts of the code such as line 42) out as constants and put them into a new `Message` class later on?